lambda/solana-memo-lambda-invoke/README.md

```diff
@@ -0,0 +1,79 @@
+# MetaKeep Lambda - Invoke Memo Program in Solana
```

Comment:
we should instead move all folders inside `solana` to `solana/transaction-signing` and then create a new folder `solana/lambda`

Reply:
created issue